doc/Learn/Navigation/Advanced/HowTo-UseTabBar.md

```diff
@@ -324,6 +324,45 @@ var builder = this.CreateBuilder(args)
     </utu:TabBar.Items>
     ```
 
+#### Using the `Navigation.Data` attached property
```

Comment:
For the Navigation.Data sections in NavigationView and TabBar, let's instead just point to the relevant existing docs at this section: https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Navigation/HowTo-NavigateInXAML.html#2-navigationdata

And these can be slimmed down to just the showing the addition of the Navigation.Data property in the xaml. Can just add a note to refer to the previously linked docs for the full setup required for data to be passes (DataViewMap and viewmodel changes)